docs/installing_deis/azure.rst

```diff
@@ -0,0 +1,121 @@
+create this certificate for you. Otherwise, you can skip to the next section.
+
+If you need to create a certificate, edit cert.conf in contrib/azure with your company's details and then run:
+
+.. code-block:: console
+
+    $ ./generate-mgmt-cert.sh
+
+Upload Management Cert
+----------------------
+
+If you haven't uploaded your management certificate to Azure (azure-cert.cer if you used the script
+in the previous section), do that now using the `management certificates tab`_ of the
+Azure portal's Settings.
+
+Also copy the Azure subscription id from this table and save it for the cluster creation script below.
+
+Create Cluster Cloud Config
+---------------------------
+
+Before we can create a cluster, we need to create a cloud config for it. The script
+``create-azure-user-data`` does this for you. This script takes the stock cluster instance config
+in ``../coreos/user-data.example``, customizes it for Azure, and inserts a unique cluster discovery
+endpoint:
+
+.. code-block:: console
+
+    $ ./create-azure-user-data $(curl -s https://discovery.etcd.io/new)
+
+This will create a azure-user-data cloud config file. We'll use this with the script in the next
+section during cluster creation.
+
+Create CoreOS Cluster
+---------------------
+
+With the management certificate and cloud config in place, we are ready to create our cluster.
+
+* Create a container called ``vhds`` within a storage account in the same region as your cluster using the Azure portal. Note the URL of the container for the cluster creation script below.
+* Choose a cloud service name for your Deis cluster for the script below. The script will automatically create this cloud service for you.
+* Choose an Azure `region`_ to use. Supply it in quotes with the ``--location`` parameter. The default is "West US".
+
+With that, let's run the azure-coreos-cluster script which will create the CoreOS cluster. Fill in the bracketed values with the values for your deployment you created above.
+
+.. code-block:: console
+
+    $ ./azure-coreos-cluster [cloud service name]
+         --subscription [subscription id]
+         --azure-cert azure-cert.pem
+         --num-nodes 3
+         --location "[location]"
+         --vm-size Large
+         --pip
+         --deis
+         --blob-container-url https://[blob container].blob.core.windows.net/vhds/
+         --data-disk
+         --custom-data azure-user-data
+
+This script will by default provision a 3 node cluster but you can increase this with the
+``--num-nodes`` parameter. Likewise, you can increase the VM size using ``--vm-size``.
+It is not recommended that you use smaller than Large (A3) sized instances.
+
+Note that for scheduling to work properly, clusters must consist of at least 3 nodes and always
+have an odd number of members. For more information, see `optimal etcd cluster size`_.
+
+
+Configure DNS
+-------------
+
+See :ref:`configure-dns` for more information on properly setting up your DNS records with Deis.
```

Comment:
I'm a bit of an Azure noob here. I'm wondering if we should add docs around where you can find the IP addresses of each machine in the Azure console. I had discovered it's in a small corner in the Virtual Machine page, so it might be nice to point that out.